server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -78,6 +91,20 @@ private static HttpServer createHttpServer(ServerConfig config, LoopResources lo
                 .maxHeaderSize(config.getMaxHeaderSize()));
     }
 
+    private static BiPredicate<HttpServerRequest, HttpServerResponse> compressableType() {
+        return (request, response) -> {
+            String contentType = response.responseHeaders()
```

Comment:
See if we can use https://netty.io/4.1/api/io/netty/handler/codec/http/HttpUtil.html#getMimeType-java.lang.CharSequence- instead. Also make sure that mimetype is always lowercased when matching.